sdk/requests/__tests__/getUserInfo.test.js

```diff
@@ -106,6 +123,7 @@ describe('getUserInfo', () => {
       redirectUri: 'redirectUri',
       postLogoutRedirectUri: 'postLogoutRedirectUri',
       accessToken: 'correctAccessToken',
+      idToken,
     });
 
     fetch.mockImplementation(() =>
```

Comment:
faltaria el expect(validateSub).not.toHaveBeenCalled();

Reply:
no está abajo? línea 144